features/keychain/api/index.d.ts

```diff
@@ -34,6 +34,7 @@ export interface KeychainApi {
   secp256k1: {
     signBuffer(params: { data: Buffer } & KeySource): Promise<Buffer>
     signBuffer(params: { data: Buffer; enc: 'der' } & KeySource): Promise<Buffer>
+    signSchnorr(params: { data: Buffer; extraEntropy?: Buffer } & KeySource): Promise<Buffer>
```

Comment:
I added this already in #174.
But, if we want to use this one, for consistency we should not add the extraEntropy param (or add for all), and we need the `tweak?: Buffer` param.

Reply:
added

Reply:
your PR can land too, it adds other api types

Reply:
Ok, I merged so you will have to rebase here. I think if we're adding `extraEntropy?` we should add for the other signBuffer signatures too.

Reply:
rebased